src/node_crypto.cc

```diff
@@ -255,13 +255,53 @@ void ThrowCryptoError(Environment* env,
                       unsigned long err,  // NOLINT(runtime/int)
                       const char* default_message = nullptr) {
   HandleScope scope(env->isolate());
```

Comment:
docs would be somewhere in here: https://github.com/nodejs/node/blob/master/doc/api/errors.md#class-error

There is no specific class for crypto errors, so docs will just have to go with generic Error but described as "only for crypto"